unity-renderer/Assets/Tutorial/Scripts/Steps/Initial/TutorialStep_GenesisGreetings.cs

```diff
@@ -63,10 +64,24 @@ public override void OnStepFinished()
         {
             base.OnStepFinished();
             tutorialController.SetTeacherCanvasSortingOrder(defaultTeacherCanvasSortOrder);
-            
+
             CommonScriptableObjects.userMovementKeysBlocked.Set(false);
         }
 
         internal void OnOkButtonClick() { stepIsFinished = true; }
+
+        private void Update()
+        {
+            if (stepIsFinished)
+                return;
+
+            Vector2Int currentCoords = CommonScriptableObjects.playerCoords.Get();
```

Comment:
i think you can subscribe to the changes of the `playerCoords` variable instead of polling the object in an update. Let me know what do you think

Reply:
Completely agree, will update this before merging